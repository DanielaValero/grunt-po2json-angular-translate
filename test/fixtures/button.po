msgid ""
msgstr ""
"Project-Id-Version: \n"
"POT-Creation-Date: \n"
"PO-Revision-Date: \n"
"Last-Translator: daniela Valero <danielavalero255@gmail.com\n"
"Language: en\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"

msgid "button/submit"
msgstr "Submit"

#msgid "button/save-change"
#msgstr "Save Change"

msgid "button/save-change"
msgid_plural "button/save-changes"
msgstr[0] "Save Change"
msgstr[1] "Save Changes"

msgid "button/register"
msgstr "Register"

msgid "button/delete-account"
msgstr "Delete Account"
