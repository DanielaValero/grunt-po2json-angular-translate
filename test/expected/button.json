{
   "button/submit": "Submit",
   "button/save-change": "{PLURAL, Save singular{Change}  plural{Changes}}",
   "button/register": "Register",
   "button/delete-account": "Delete Account"
}